src/Fixer/PhpTag/NoShortEchoTagFixer.php

```diff
@@ -26,7 +26,17 @@
      */
     public function isCandidate(Tokens $tokens)
     {
-        return $tokens->isTokenKindFound(T_OPEN_TAG_WITH_ECHO);
+        return $tokens->isTokenKindFound(T_OPEN_TAG_WITH_ECHO)
+            /*
+             * HHVM parses '<?=' as T_ECHO instead of T_OPEN_TAG_WITH_ECHO
+             *
+             * @see https://github.com/facebook/hhvm/issues/4809
```

Comment:
>You can't use phpdoc annotations when this isn't T_DOC_COMMENT.

there are no rules for comment content we follow, there is no need also, this PR proves that